packages/theme/src/theme/_type.scss

```diff
@@ -0,0 +1,19 @@
+h1,
+.h1 {
+	font-weight: bold;
+	color: $black;
```

Comment:
You have
* `$headings-color` variable set to `inherit`
* `$headings-font-weight` set to 600

You can apply a default that is meaningful for the headings and override only what differs inn this file

Reply:
what is the default ? I dont see any 'default' in the guideline

Reply:
I don t see any default meaningful in the guideline

Reply:
Not a `default` but something meaningful. It's about implementation, not specification.
We get to decide how to do it. And the current implem has 2 vars that are set, but never really used because they are overridden by this file.

So let's say `$black` and `600` and you override them as needed